src/GenAPI/Microsoft.DotNet.GenAPI/INamedTypeSymbolExtension.cs

```diff
@@ -71,16 +71,22 @@ private static SyntaxList<AttributeListSyntax> FromAttributeData(IEnumerable<Att
         }
 
         // Build dummy field from a type, field name, and attribute list.
-        private static SyntaxNode CreateDummyField(string typ, string fieldName, SyntaxList<AttributeListSyntax> attrs) =>
-            SyntaxFactory.FieldDeclaration(
+        private static SyntaxNode CreateDummyField(string typ, string fieldName, SyntaxList<AttributeListSyntax> attrs, bool isReadonly)
+        {
+            List<SyntaxToken> modifiers = new List<SyntaxToken> { SyntaxFactory.Token(SyntaxKind.PrivateKeyword) };
```

Comment:
If we're gonna use `readonly` everywhere, then we don't need the `if` statement. We can just pass in
```C#
WithModifiers(SyntaxFactory.TokenList(new[] { SyntaxFactory.Token(SyntaxKind.PrivateKeyword), SyntaxFactory.Token(SyntaxKind.ReadOnlyKeyword) })
```

Reply:
I checked the CCI-based GenAPI and it does not always include the `readonly` keyword. But, I believe we could use this keyword in all cases. @ViktorHofer , what do you think?